src/resources/language/_language-fr.yml

```
@@ -51,3 +51,22 @@ crossref-eq-prefix: "Équation"
 crossref-lof-title: "Liste des Figures"
 crossref-lot-title: "Liste des Tables"
 crossref-lol-title: "Liste des Listings"
+
+listing-page-order-by: "Commander par"
```

Just a comment on the translation here. I don't think this is the right one 🤔 

Is `Order By` meaning `Sort by` ? 

That would be in French : "Trier par " 
Or less common "Ordonner par". 

```suggestion
listing-page-order-by: "Trier par"
```